code/game/objects/structures/tables_racks.dm

```diff
@@ -583,6 +589,74 @@
 	return ..()
 
 /*
+ * Glass
+ */
+
+/obj/structure/table/glass
+	name = "glass table"
+	desc = "A standard table with a fine glass finish."
+	icon_state = "glass_table"
+	parts = /obj/item/weapon/table_parts/glass
+
+
+/obj/structure/table/glass/attackby(obj/item/W as obj, mob/user as mob, params)
```

Comment:
Really? `params` is just a function parameter, unless you're calling `..()` it doesn't do anything, how weird.